nomos-services/data-availability/src/auth/mod.rs

```diff
@@ -0,0 +1,11 @@
+pub mod mock;
```

Comment:
We should probably feature gate this (not necessarely in this PR) (?)

Reply:
We have more mocks in core unrelated to DA, we could have one PR to feature gate them all.